content/guides/react/hooks.mdx

```diff
@@ -0,0 +1,12 @@
+---
+title: Primer React hooks
+---
+
+- [`useSafeTimeout`](/guides/react/hooks/use-safe-timeout)
+- [Anchored position](/guides/react/hooks/anchored-position)
```

Comment:
The mixed formatting feels a little odd. I'd keep them all regular text and just use the hook name.